File: Data Structures & Algorithms/longest-repeating-substring-with-replacement/submission-1.cpp
class Solution {
public:
    int characterReplacement(string s, int k) {
        vector<int>hash(27,0);
        int n = s.size();
        int left = 0,right =0;
        int mf = 0;
        int ans = 0;
        while(right<n){
            if(right-left-mf<=k){
                ans = max(ans,right-left);
                hash[s[right]-'A']++;
                mf = max(mf,hash[s[right]-'A']);
                right++;
            }
            else{
                hash[s[left]-'A']--;
                left++;
            }
        }
        if(right-left-mf<=k) ans = max(ans,right-left);
        return ans;
       
        
    }
};

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-1.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        q=deque()
        if not root:
            return []
        q.append(root)
        ans=[]
    
        while q:
            s = len(q)
            temp=[]
            for _ in range(s):
                node = q.popleft()
                temp.append(node.val)
                q.append(node.left) if node.left else _
                q.append(node.right) if node.right else _
            ans.append(temp)



        return ans
